webapp/src/components/modals/create_issue/create_issue_form.tsx

```diff
@@ -136,7 +136,8 @@ export default class CreateIssueForm extends React.PureComponent<Props, State> {
         });
 
         let fields = {
-            ...this.state.fields,
+            summary: this.state.fields.summary,
+            description: this.state.fields.description,
```

Comment:
Wasn't it being happening with the previous code where `state.fields` is being expanded using the spread operator? What's the need for this change?

Reply:
actually, there are more than in the state. fields that we don't want that's why we are only taking the values we need.